src/main/java/de/medizininformatikinitiative/fhir_data_evaluator/PopulationCount.java

```diff
@@ -1,19 +1,23 @@
 package de.medizininformatikinitiative.fhir_data_evaluator;
 
-import org.hl7.fhir.r4.model.CodeableConcept;
 import org.hl7.fhir.r4.model.MeasureReport;
 
 public record PopulationCount(HashableCoding code, int count) {
```

Comment:
Pleas add docs and just call it Population. A population is just the number of it's members. We will not need to handle the actual members here.